pipeline/inputs/kafka.md

```diff
@@ -12,6 +12,8 @@ This plugin uses the official [librdkafka C library](https://github.com/edenhill
 | client\_id | Client id passed to librdkafka. | |
 | group\_id | Group id passed to librdkafka. | fluent-bit |
 | poll\_ms | Kafka brokers polling interval in milliseconds. | 500 |
+| Chunk\_Size  | Specify the size of buffer to store the incoming kafka messages. When this parameter is specified, do not poll them from brokers at once over the specified size and deferred to consume left messages from there at the next polling cycle. If not set, _Chunk\_Size_ is not effective. | |
```

Comment:
Can you clarify the description a bit here?
What happens if this is not specified?

I think this is saying you define a chunk size that will read X number of kafka messages until it is full? Once full it blocks until Fluent Bit reads them then continues pulling more messages - right?

Reply:
When not specified, the polling operation is running until events are fully consumed on the subscribed topics.
When specified, the polling operation is running until the limit and halt once it'll be reached the buffer_chunk_size.
The motivation of this parameter is preventing for greedy polling operations in in_kafka.

Reply:
```suggestion
| Chunk\_Size  | By default, greedy polling is carried out so all events are fully consumed on all subscribed topics. This option allows you to specify a limit for polling so it stops once it hits this size. | |
```